00_Base/src/model/DTO/AdminUpdateCredentialsRequestDTO.ts

```diff
@@ -0,0 +1,37 @@
+import { IsEnum, IsNotEmpty, IsString } from 'class-validator';
+import { CredentialsRoleDTO } from './CredentialsRoleDTO';
+import { AdminCredentialsRequestDTO } from './AdminCredentialsRequestDTO';
+import { CountryCode } from '../../util/util';
+
+export class AdminUpdateCredentialsRequestDTO extends AdminCredentialsRequestDTO {
+  @IsEnum(CountryCode)
+  @IsNotEmpty()
+  mspCountryCode!: CountryCode;
```

Comment:
Ah, I was wondering if we should make an enum for this, but also seeing that I made exact same model in my code, we should probably just have a `ServerClientCountryCodePartyId` class or something 👍

Reply:
@lydiazcheng @elliot-sabitov do we need to make any code changes to align this PR with [this PR](https://github.com/citrineos/citrineos-ocpi/pull/91#discussion_r1691955321)?

Reply:
we dont have to do that for this PR though